spectrocloud/resource_cluster_group_test.go

```diff
@@ -32,6 +32,11 @@ func prepareClusterGroupTestData() *schema.ResourceData {
 			"host_dns":    "https://test.dev.spectro.com",
 		},
 	})
+	d.Set("cluster_profile", []map[string]interface{}{
```

Comment:
@SivaanandM please handle set error in the test too.

Reply:
addressed